policy-design-lab/pdl-frontend
# #301: add null value check for new csp data

This PR is for #301. The new CSP backend update requires a minor adjustment for null handling in the CSP table. This PR addresses that and ensures the CSP page functions correctly.

## How to Test
Open the CSP page in the development environment. You should see the page load successfully.
